nginx-config.yml.sample

```diff
@@ -4,7 +4,14 @@ instances:
     - name: nginx-server-metrics
       command: metrics
       arguments:
+          # If you're using ngx_http_api_module be certain to use the full path up to and including the version number
           status_url: http://127.0.0.1/status
+	      # Name of Nginx status module OHI is to query against. discovery | ngx_http_stub_status_module | ngx_http_status_module | ngx_http_api_module
+	      status_module: discovery
```

Comment:
is this line proper formatted?

Reply:
Yes

Reply:
There are tabstop characters on those two lines, causing a YAML syntax error when the agent tries to parse the config.